static/src/javascripts/projects/common/modules/commercial/track-ad.js

```diff
@@ -0,0 +1,30 @@
+define([
+    'Promise',
+    'common/utils/mediator'
+], function(Promise, mediator) {
+    return trackAd;
+
+    function trackAd(id) {
```

Comment:
Yes indeed. That what I'd like to do next, but it's a bit more work—trackAd is stateful (this is why `Promise` is much better than `mediator`).